superset/config.py

```diff
@@ -1453,6 +1455,7 @@ def EMAIL_HEADER_MUTATOR(  # pylint: disable=invalid-name,unused-argument
             "data:",
             "https://apachesuperset.gateway.scarf.sh",
             "https://static.scarf.sh/",
+            "https://avatars.slack-edge.com",
```

Comment:
I'm not sure anymore if the ASF will care about this or not.

Reply:
They shouldn't, especially as these are just default conf settings, and we don't store anything. Personally can't think of a rule anyone would be breaking by turning this on, plus it's off by default.

Reply:
I think it's that when any user traffic hits a 3rd party service, they could get IPs/analytics from it, in which case they'll want to have a DPA in place. They might already have one with Slack, which would make this a moot point, but if not, we can probably get one one, if this is even a concern.

Reply:
Think I should tie this to the feature flag?

Reply:
Ok I made SLACK_ENABLE_AVATARS = False by default (at least for now), and commented the `TALISMAN_CONFIG` changes out, with a note about de-commenting the line if you set SLACK_ENABLE_AVATARS to True. Altered the UPDATING.md comment to reflect that.

Reply:
Made all defaults more conservative,  `SLACK_ENABLE_AVATARS` is False by default and users have to alter their `TALISMAN_CONFIG` manually, resolving this thread.